contrib/sssd.spec.in

```diff
@@ -1039,7 +1039,7 @@ install -D -p -m 0644 %{SOURCE1} %{buildroot}%{_sysusersdir}/sssd.conf
 
 %if %{use_sssd_user}
 %pre common
-! getent passwd sssd >/dev/null || usermod sssd -d /run/sssd >/dev/null || true
+! getent passwd sssd >/dev/null || usermod sssd -d /run/sssd 2>&1 >/dev/null || true
```

Comment:
What is the goal of this change? Removing any message in `stderr` or moving them to `stdout`? The latter is what it is doing. To achieve the first one, you need to invert the redirections: ` usermod sssd -d /run/sssd >/dev/null 2>&1`

Reply:
Thanks, fixed.